control/joy_controller/README.md

```diff
@@ -62,3 +62,24 @@
 | Autoware Disengage   | Y                     |
 | Vehicle Engage       | PS                    |
 | Vehicle Disengage    | Right Trigger         |
+
+## DS4 Joystick Key Map
+
+| Acceleration         | R2                                       |
+| -------------------- | ---------------------------------------- |
+| Brake                | □ or Right Stick Up Down or Left Trigger |
```

Comment:
Maybe this is correct? :thinking: 

```suggestion
| Action               | Button                                   |
| -------------------- | ---------------------------------------- |
| Acceleration         | R2, ×, or Right Stick Up                 |
| Brake                | L2, □, or Right Stick Down               |
```

Reply:
Could you also fix the table header of the P65's keymap?
![image](https://user-images.githubusercontent.com/31987104/189787156-76487de1-8e78-47b3-99db-67cd8972cb59.png)

Reply:
Done